tests/CMakeLists.txt

```diff
@@ -106,9 +113,10 @@ LIST(SORT _tests)
 FOREACH(_test ${_tests})
   GET_FILENAME_COMPONENT(_test ${_test} NAME_WE)
 
-  CHECK_LINEAR_ALGEBRA(${CMAKE_CURRENT_SOURCE_DIR}/${_test}.prm)
+  SET(_use_test ON)
+  SHOULD_ENABLE_TEST(${CMAKE_CURRENT_SOURCE_DIR}/${_test}.prm)
 
-  IF("${_use_test}" STREQUAL "1")
+  IF(_use_test)
```

Comment:
Ugh, this is what PARENT_SCOPE does -- know about the name of a particular variable in a different scope? I sure hate people designing programming languages who have no idea how to do it...